### DISCUSSION TITLE
更新3.28版本后，阿里云open无法同时挂载资源盘和备份盘

### DISCUSSION THREAD

**Discussion by sdliangl** [AlistGo/alist#5283] [General]
在挂载设置里面一个设置资源盘，一个设置备份盘，但两个要么同时都是资源盘，要么同时都是备份盘。

**Comment by anwen-anyi** [comment]
你可以添加两个存储吖 一个类型选择资源库 一个类型选择备份盘

**Reply by chenshuai19960520** [reply]
为什么我类型我选择了 资源库 但是出来还是备份盘的内容 没有出现对应资源库的目录

**Comment by sdliangl** [comment]
我就这么干的！不得行



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: 安稳 ***@***.***&gt;
发送时间: 2023年9月25日 19:02
收件人: alist-org/alist ***@***.***&gt;
抄送: sdliangl ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [alist-org/alist] 更新3.28版本后，阿里云open无法同时挂载资源盘和备份盘 (Discussion #5283)

**Reply by sparkssssssss** [reply]
我是这样用的,没问题,一个资源一个备份
